src/schema/basic_slots.yaml

```diff
@@ -81,9 +81,11 @@ slots:
       - prov:endedAtTime
 
   git_url:
+    description: The url that points to the exact github location of a workflow.
```

Comment:
thanks. let's discuss this with other people who run workflows in the near future.